packages/eddsa-poseidon/package.json

```diff
@@ -60,6 +60,7 @@
     "dependencies": {
         "@zk-kit/baby-jubjub": "1.0.3",
         "@zk-kit/utils": "1.2.1",
+        "blake-hash": "^2.0.0",
         "blakejs": "^1.2.1",
```

Comment:
- Does the blakejs library provide a blake1 hash which could be used here instead?
- Alternately, have you checked whether the blakejs library might have the same error that this PR is fixing, and need its own update for the blake2b use case?

Reply:
> Does the blakejs library provide a blake1 hash which could be used here instead?

`blakejs` should only provide the following implementations: BLAKE2b and BLAKE2s.

> Alternately, have you checked whether the blakejs library might have the same error that this PR is fixing, and need its own update for the blake2b use case?

Let me tag @thogiti for this question.